chapter_natural-language-processing-pretraining/glove_vn.md

```diff
@@ -39,8 +41,14 @@ Denote multiplicity of element $j$ in multiset $\mathcal{C}_i$ as $x_{ij}$: it i
 As a result, the loss function of the skip-gram model can be expressed in a different way:
 -->
 
-*dịch đoạn phía trên*
-
+Với từ $w_i$, nó có thể xuất hiện trong tập dữ liệu nhiều lần.
+Ta gom tất cả các từ ngữ cảnh mỗi lần khi $w_i$ là từ trung tâm và giữ các từ trùng lặp, và ký hiệu chúng là tập bội $\mathcal{C}_i$.
+Số lượng của một phần tử trong tập bội được gọi là bội số của phần tử đó. 
+Chẳng hạn, giả sử rằng từ $w_i$ xuất hiện hai lần trong tập dữ liệu:
+Cửa sổ ngữ cảnh khi hai từ $w_i$ đó trở thành từ trung tâm trong chuỗi văn bản chứa các chỉ số từ ngữ cảnh $2, 1, 5, 2$ và $2, 3, 2, 1$.
+Thì, tập bội $\mathcal{C}_i = \{1, 1, 2, 2, 2, 2, 3, 5\}$, ở đây bội số của phần tử số 1 là 2, bội số của phần tử số 2 là 4, và bội số của phần tử số 3 và 5 đều là 1.
+Ký hiệu bội số của phần tử $j$ trong tập bội $\mathcal{C}_i$ là $x_{ij}$: nó là số từ $w_j$ trong tất cả cửa sổ ngữ cảnh để từ trung tâm $w_i$ ở trong toàn bộ tập dữ liệu.
```

Comment:
```suggestion
Ký hiệu bội số của phần tử $j$ trong tập bội $\mathcal{C}_i$ là $x_{ij}$: nó là số lần từ $w_j$ có trong cửa sổ ngữ cảnh của từ trung tâm $w_i$ trong toàn bộ tập dữ liệu.
```

Reply:
Câu này mình nghĩ nên sửa lại một chút nữa cho dễ đọc hơn là:
"Ký hiệu bội số của phần tử $j$ trong tập bội $\mathcal{C}_i$ là $x_{ij}$: nó là số lần từ $w_j$ có trong cửa sổ ngữ cảnh mà từ $w_i$ làm trung tâm trong toàn bộ tập dữ liệu."

Reply:
```suggestion
Ký hiệu bội số của phần tử $j$ trong tập bội $\mathcal{C}i$ là $x{ij}$: nó là số lần từ $w_j$ có trong cửa sổ ngữ cảnh mà từ $w_i$ làm trung tâm trong toàn bộ tập dữ liệu.
```